src/components/global-toolbar/index.js

```diff
@@ -70,7 +71,7 @@ export const GlobalToolbar = ({
             className={
               'pipeline-menu-button--settings pipeline-menu-button--large'
             }
-            onClick={() => onToggleSettingsModal(true)}
+            onClick={() => onToggleSettingsModal(!visible.settingsModal)}
```

Comment:
Thanks, so the general rule is that modal should be on top of everything! Because one of the things I realised was plotly modal overlapped the global sidebar but others didn't.

Reply:
Yes I think the modal should sit on top of all content :)